exporters/etw/include/opentelemetry/exporters/etw/etw_properties.h

```diff
@@ -222,7 +222,8 @@ class PropertyValue : public PropertyVariant
         break;
       }
       case common::AttributeType::kTypeString: {
-        PropertyVariant::operator=(nostd::string_view(nostd::get<nostd::string_view>(v)).data());
+        PropertyVariant::operator=
+            (std::string{nostd::string_view(nostd::get<nostd::string_view>(v)).data()});
```

Comment:
Nit - Can we change the method comments as this method changes non-owning common::AttributeValue to owning PropertyValue.